typescript/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
```

Comment:
No, to tsconfig only defines the structure and compilerOptions for this module. Users should be able to import the module fine as long as our type definitions are correct, regardless if their project settings match these or not.

In my opinion this is still a non-breaking change, as long as users used the interface correctly. I didn't change anything major at the tests (as I said, the Deletion was unintentional) and they still pass.